source/security.txt

```diff
@@ -16,12 +16,12 @@ Secure Your App
 .. toctree::
    :titlesonly:
    :hidden:
- 
+
    Configure Network Security </security/network>
+   Control Developer Access </security/developer-access>
```

Comment:
This is just a nit, and I'm not sure whether you covered this in your tree test, but I wonder if this ToC item/page name should be "Configure Developer Permissions"? I think that better parallels "Configure Network Security", and "control" just rubs me the wrong way for some reason. If we do want to emphasize the restriction aspect of this, maybe we could consider a name like "Limit Developer Access" or "Restrict Developer Access"?